README.md

```diff
@@ -74,17 +74,17 @@ files:
 
 ### CSV Data Files
 
-One CSV file can only store one type of vertex or edge. Vertices and edges of different schema should be stored in different files.
+One CSV file can only store one type of vertex or edge. Vertices and edges of the different schema should be stored in different files.
 
 * `path`: **Required**. Specifies the path where the CSV data file is stored. If a relative path is used, the path and directory of the current configuration file are spliced.
 * `failDataPath`: **Required**. Specifies the file to insert the failed data output so that the error data is appended later.
 * `batchSize`: **Optional**. Specifies the batch size of the inserted data, the default value is 128.
-* `type & csv`:  **Required**. Specifies the file type. Currently only CSV is supported. You can specify whether to include the header and the inserted and deleted labels in the CSV file.
+* `type & csv`:  **Required**. Specifies the file type. Currently, only CSV is supported. You can specify whether to include the header and the inserted and deleted labels in the CSV file.
   * `withHeader`: The default value is false, the format of the header is described below.
   * `withLabel`: The default value is false, the format of the label is described below.
 
 * `schema`: **Required**. Describes the metadata information of the current data file. The schema.type has only two values: vertex and edge.
-  * When type is specified as vertex, details should be described in vertex field.
```

Comment:
When the type is specified as vertex, details must be described in the **vertex** field.